lib/bitcrowd_ecto/assertions.ex

```diff
@@ -24,6 +24,7 @@ defmodule BitcrowdEcto.Assertions do
 
   @moduledoc since: "0.1.0"
 
+  alias Ecto.Changeset
```

Comment:
btw @ammancilla "conventional" order is `use-import-require-alias` https://github.com/christopheradams/elixir_style_guide#modules

but we're not following it too closely 🤷 just mentioning it because I saw you reordering things

Reply:
I really need to make time to read the whole guideline. Thanks for sharing again.

Reply:
> Or is there?

probably disable because it's annoying :D